src/dodal/beamlines/i22.py

```diff
@@ -35,3 +36,81 @@ def it(
         fake_with_ophyd_sim,
         directory_provider=get_directory_provider(),
     )
+
+
+def slits_1(
+    wait_for_connection: bool = True,
+    fake_with_ophyd_sim: bool = False,
+) -> Slits:
+    return device_instantiation(
+        Slits,
+        "slits-01",
+        "-AL-SLITS-01:",
+        wait_for_connection,
+        fake_with_ophyd_sim,
+    )
```

Comment:
Could: I would still want the `slits_x()` interface but no reason you can't hide it:
```python
def _generic_slit(slit_num, wait_for_connection, fake):
        return device_instantiation(
        Slits,
        f"slits-0{slit_num}",
        f"-AL-SLITS-0{slit_num}:",
        wait_for_connection,
        fake_with_ophyd_sim,
    )

def slits_1(...):
    return _generic_slit(1, ...)
...
```

Reply:
Good idea, is there currently anywhere sensible to put `_generic_slit` so that it can be shared between the `i22` and `p38` modules?

Reply:
Hmm.. I guess `devices.util.something`?

Reply:
I ended up putting it in a private module in `beamlines` because otherwise we were importing `device_instantiation` from `beamline.beamline_utils` into `devices.util.something`, then importing that back into `beamlines.i22`, which I could lead to circular import troubles at some point. But let me know what you think, happy to change it